modules/oe_theme_content_tender/src/Plugin/PageHeaderMetadata/TenderContentType.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_theme_content_tender\Plugin\PageHeaderMetadata;
+
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\oe_theme_helper\Plugin\PageHeaderMetadata\NodeViewRoutesBase;
+use Drupal\oe_content_tender\TenderNodeWrapper;
+
+/**
+ * Page header metadata for the OpenEuropa Call for tender content type.
+ *
+ * @PageHeaderMetadata(
+ *   id = "oe_tender_content_type",
+ *   label = @Translation("Metadata extractor for the OE Call for Tender content type"),
+ *   weight = -1
+ * )
+ */
+class TenderContentType extends NodeViewRoutesBase {
+
+  use StringTranslationTrait;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function applies(): bool {
+    $node = $this->getNode();
+
+    return $node && $node->bundle() === 'oe_tender';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getMetadata(): array {
+    $metadata = parent::getMetadata();
+    $node = $this->getNode();
+    if (!($node->get('oe_summary')->isEmpty())) {
```

Comment:
Extra () for negation is not needed. Move the code from the if outside and if oe_summary is empty return the $metadata from the parent method. Add metas info before checking if oe_summary is empty.